crates/full-node/sov-sequencer/src/preferred/block_executor.rs

```diff
@@ -98,14 +98,13 @@ impl<S: Spec> RollupBlockExecutorError<S> {
 type StateRootReceiver<S> =
     oneshot::Receiver<(RollupHeight, <<S as Spec>::Storage as Storage>::Root)>;
 
-pub struct RollupBlockExecutorConfig<S: Spec, Rt: Runtime<S>> {
-    pub config: SequencerConfig<S::Address, PreferredSequencerConfig>,
+#[derive(Clone)]
+pub struct RollupBlockExecutorConfig<S: Spec> {
     pub da_address: <S::Da as DaSpec>::Address,
     pub shutdown_notifier: Sender<()>,
-    pub state_root_request_sender: tokio::sync::mpsc::Sender<StateRootComputeRequest<S>>,
     pub shutdown_receiver: watch::Receiver<()>,
     pub shutdown_sender: watch::Sender<()>,
-    pub startup_transaction_cache_writer: Option<TxResultWriter<S, Rt>>,
+    pub state_root_request_sender: Sender<StateRootComputeRequest<S>>,
```

Comment:
This is down to preference but I like keeping the module name, i.e. `mpsc::Sender` since tokio has multiple different channels with identical looking Sender and Receiver structs